docs/source/developer_guide/core/transport.rst

```diff
@@ -3,369 +3,59 @@ Transport plugins
 
 .. toctree::
    :maxdepth: 2
-   
-This chapter describes the generic implementation of a transport plugin.
-The currently implemented are the local and the ssh plugin.
-The local plugin makes use only of some standard python modules like os and shutil.
-The ssh plugin is a wrapper to the library paramiko, that you installed with AiiDA.
 
-A generic set of tests is contained in plugin_test.py, while plugin-specific tests are written separately.
+The term `transport` in AiiDA, refers to a utility that the engine can use to perform certain operations on the machines where its processes are running.
```

Comment:
This is a very generic sentence, I don't find it too clear. Maybe change to something like "...to a class that the engine uses to perform operations (copy of files, execution of commands) on local or remote machines where its CalcJobs are submitted."? Or something similar?

Reply:
Adapted